src/CoreGraphics/CGColorConverter.cs

```diff
@@ -26,8 +26,8 @@ namespace CoreGraphics {
 #if !NET
 	[TV (9,2)]
 	[iOS (9,3)]
-	[Obsoleted (PlatformName.TvOS, 10,0, message: "Replaced by 'GColorConversionInfoTriple'.")]
-	[Obsoleted (PlatformName.iOS, 10,0, message: "Replaced by 'GColorConversionInfoTriple'.")]
+	[Obsoleted (PlatformName.TvOS, 10,0, message: "Replaced by 'CGColorConversionInfoTriple'.")]
```

Comment:
This is for legacy Xamarin, it's the Obsolete attributes just below that should be updated instead.